kubernetes/samples/scripts/domain-lifecycle/stopDomain.sh

```diff
@@ -66,13 +66,21 @@ function initialize {
 
   validateKubernetesCliAvailable
   validateJqAvailable
+  validateYqAvailable
   failIfValidationErrors
 }
 
 initialize
 
 # Get the domain in json format
-domainJson=$(${kubernetesCli} get domain ${domainUid} -n ${domainNamespace} -o json)
+domainJson=$(${kubernetesCli} get domain ${domainUid} -n ${domainNamespace} -o json --ignore-not-found)
+
+if [ -z "${domainJson}" ]; then
+  printError "Domain resource for domain '${domainUid}' not found in namespace '${domainNamespace}'. \
+    This script requires that the introspector job for the specified domain ran \
```

Comment:
This information is not needed. The first statement ( Domain resource for domain '${domainUid}' not found in namespace '${domainNamespace} ) is sufficient convey the message. Should we put internal detail about introspector job here ?

Reply:
Done.